docs/firehose.MD

```diff
@@ -80,7 +80,7 @@ The requests can be batched so that multiple prefixes may be requested.
     id, # int
     
     # The block hash as of which we request the data
-    block_hash, # array of 32 bytes
+    block_hash, # 32 bytes
```

Comment:
`array of` seems implementation-specific. In python it will just be a 32-length `bytes` value. Array probably makes sense in other languages. I don't *think* leaving it at `32 bytes` is ambiguous here, since the RLP definition of a series of bytes is pretty straightforward.